spinalcordtoolbox/deepseg/core.py

```diff
@@ -85,16 +85,20 @@ def fill_holes(nii_seg):
     return nii_seg
 
 
-def segment_nifti(fname_image, folder_model, param={}):
+def segment_nifti(fname_image, folder_model, fname_prior=None, param=None):
```

Comment:
Changed `param={}` to `param=None` because setting default argument value to a mutable variable is not a [good practice](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html)